pkg/testing/runner/sles.go

```diff
@@ -0,0 +1,112 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Do we still need the SLES runner?

Reply:
No, we don't really need it.  I was using it for my testing because the SSH connections had failures often enough to make it easy to see that the SSH changes were working.  Anyway, I can remove it.